README.md

```diff
@@ -112,6 +112,10 @@ Use Scala's build tool
 ([sbt](https://www.scala-sbt.org/download.html)) to build and run
 Janus.
 
+#### Install Node
+
+The version is specified in `.nvmrc`. Dependencies will be installed as part of the sbt run command.
```

Comment:
They will the *first* time (based on the proxy of the `node_modules` directory not existing). Is it worth mentioning that `npm install` might be required while working on the client-side? No dramas either way!